ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa-4.12-nightly.yaml

```diff
@@ -81,7 +81,12 @@ tests:
     cluster_profile: aws-perfscale-qe
     env:
       CHANNEL_GROUP: nightly
+      COMPUTE_MACHINE_TYPE: m5.2xlarge
+      CONFIG: config/classic-rosa.yml
```

Comment:
@krishvoor I think you'll need to add support to override this env variable in https://github.com/openshift/release/blob/master/ci-operator/step-registry/openshift-qe/ingress-perf/openshift-qe-ingress-perf-commands.sh